### PR TITLE
Add context to close events

### PR DESCRIPTION
**Pull request by wmaethner**
Added context to the current close events (patient and study). @NiklasSvenzen I know you mentioned also adding context to the logout events, that would be the user right? If we added that what would the resource be (we don't have anything like that in any of the other events), or since we have already done validation of this subscription based on the topic, does the topic already have enough user context to handle the logout events?

### CODE REVIEW

File: docs/specification/May2019Ballot.md [hunk 1]
```diff
@@ -428,10 +428,33 @@ Context | Optionality | FHIR operation to generate context|  Description
...
+      }
+    }
+  ]
+}
 ```
 
-No context.
+Context | Optionality | FHIR operation to generate context|  Description
+--- | --- | --- | ---
+`patient` | Required |  `Patient/{id}?\_elements=identifier` | FHIR Patient resource describing the patient whose chart is currently in context..
```
Comment (lbergnehr) [inline docs/specification/May2019Ballot.md]:
```suggestion
`patient` | Required |  `Patient/{id}?_elements=identifier` | FHIR Patient resource describing the patient whose chart is currently in context..
```

File: docs/specification/May2019Ballot.md [hunk 2]
```diff
@@ -428,10 +428,33 @@ Context | Optionality | FHIR operation to generate context|  Description
...
+`encounter` | Optional | `Encounter/{id}?\_elements=identifier` | FHIR Encounter resource in context in the newly opened patient's chart.
```
Comment (lbergnehr) [inline docs/specification/May2019Ballot.md]:
```suggestion
`encounter` | Optional | `Encounter/{id}?_elements=identifier` | FHIR Encounter resource in context in the newly opened patient's chart.
```

**Comment by wmaethner** [comment]
Good catch @lbergnehr. Found a few other instances with that same typo since I basically just copied the context from the other events, so I went ahead and fixed those as well. Re-committing and requesting review.

File: docs/specification/May2019Ballot.md [hunk 3]
```diff
@@ -428,10 +428,33 @@ Context | Optionality | FHIR operation to generate context|  Description
...
+{
+  "context": [
+    {
+      "key": "patient",
+      "resource": {
+        "resourceType": "Patient",
+        "id": "ewUbXT9RWEbSj5wPEdgRaBw3",
+        "identifier": [
+          {
+            "system": "urn:oid:1.2.840.114350",
```
Comment (daliboz) [inline docs/specification/May2019Ballot.md]: Are any of these identifiers that would have a URL system? If no, should we add the identifier types as examples here? EG: MRN? 

```
"identifier": [
   {
       "type": {
            "coding": [
                  {
                      "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
                      "value": "MR",
                      "display": "Medication Record Number"
                   },
                  "text": "MRN"
              ],
          }
           ....
     }, ...
],...
```

File: docs/specification/May2019Ballot.md [hunk 4]
```diff
@@ -538,10 +561,49 @@ Context | Optionality | FHIR operation to generate context|  Description
...
+    },
+    {
+      "key": "study",
+      "resource": {
+        "resourceType": "ImagingStudy",
+        "id": "8i7tbu6fby5ftfbku6fniuf",
+        "uid": "urn:oid:2.16.124.113543.6003.1154777499.30246.19789.3503430045",
+        "identifier": [
+          {
+            "system": "7678",
```
Comment (daliboz) [inline docs/specification/May2019Ballot.md]: Similar comment as above, except I think this is syntactically invalid. This I would expect to probably be an oid system like above? Not sure there's a good "type" that would apply here.

Comment (wmaethner) [reply docs/specification/May2019Ballot.md]: I don't know enough about the FHIR identifiers to determine what should go here. You have any ideas?

**Pull request merged**